### ISSUE TITLE
Not working with new VSCode "floating editor windows" feature

### ISSUE THREAD

**Issue by TonyGravagno** [PejmanNik/vscode-autoScroll#32]
VSCode v1.85.1 over Windows 10 with AutoScroll 1.1.0

The "AS" toggle button is still in the footer of the main VSCode window:
![image](https://github.com/PejmanNik/vscode-autoScroll/assets/499473/3c315ead-fec9-407a-8ee2-cd9d7618958c)

But is not yet in the footer of new windows that are drag/dropped out of the main container.
![image](https://github.com/PejmanNik/vscode-autoScroll/assets/499473/ff3ef38a-3966-470d-8139-eb6edfa183dd)

This could be due to a bug in the new VSCode feature, or maybe that apps like this need to be enhanced to support the new feature.
https://github.com/microsoft/vscode/issues?q=is:open+is:issue+label:workbench-auxwindow

Either way, thanks for any effort, and thanks for this extension.

**Comment by PejmanNik** [comment]
Yes, unfortunately, i couldn't find any API that lets extensions add status bar items to the floating windows, for now, you can enable it by pressing `F1` and then `auto Scroll: enable`
